debian/control

```diff
@@ -13,9 +13,11 @@ Build-Depends:
  bpftool,
  ronn,
  libbpf-dev,
+ libelf-dev,
  libffi-dev,
  libxdp-dev,
  linux-source,
+ zlib1g-dev,
```

Comment:
Why these two are needed when the package successfully builds on Debian 12 amd64?